app/index.js

```diff
@@ -45,6 +46,7 @@ document.title = 'Dat Desktop | Welcome'
 datMiddleware
   .loadFromDisk()
   .then(function () {
+    document.body.appendChild(datIcons())
```

Comment:
I think it might be better to have a comment here, explaining why this is outside of the react rendering.

Something like:

```javascript
// To reduce the load of the react rendering processwe simply add the icons to the dom.
```

Reply:
@martinheidegger san
This PR have a two purpose.
1: React-Tree everything need the "CLEAN". ( ENV-Components is not required for React-Tree ! )
2: And, I think a next task is "CLEAN-ROUTING". The "Sprite" component is obstacle for that task. ( In every routing require the render a "Sprite" component. but it is waste and dirty. )

Reply:
How about following comment then:

```javascript
.then(function addGlobalComponents () {
  // Adding global components only once to the DOM.
  document.body.appendChild(datIcons())
})
.then(function () {
```

Reply:
I did added.
Thank you for recommendation.